### ISSUE TITLE
data shuffling in data loader

### ISSUE THREAD

**Issue by homelifes** [ruotianluo/self-critical.pytorch#153]
Hi @ruotianluo . Thanks for your code.
I have seen from `dataloader.py` that you aren't shuffling the training data before loading it to the network. Would that affect the performance?

**Comment by ruotianluo** [comment]
I do shuffle

**Comment by ruotianluo** [comment]
And FYI, shuffling is better from my experience.
